pset-functions.js

```diff
@@ -76,6 +110,45 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+
+function scrabbleScore(word) {
+//object of all letters and their corresponding value for Scrabble game
+  letterValues = {'A':1,'E':1,'I':1,'O':1,'U':1,'L':1,'N':1,'R':1,'S':1,'T':1,'D':2,'G':2,'B':3,'C':3,'M':3,'P':3,'F':4,'H':4,'V':4,'W':4,'Y':4,'K':5,'J':8,'X':8,'Q':10,'Z':10}
+
+//variable for sum of letters in word
+let wordScore = 0
+
+//string given in lower case so I am using the toUpperCase method to convert the string to UPPERCASE letters
+
+let lettersToScore = word.toUpperCase().split('')
+
+//Using the split method to take every letter in the string into an array
+//lettersToScore = lettersToScore.split('')
+
+//console.log(lettersToScore)
+
+//loop through string and sum the value of the letter scores
+
+for (let i = 0; i<lettersToScore.length;i++) {
+  let letter = lettersToScore[i]
+  wordScore = wordScore + letterValues[letter]
+  //console.log(wordScore)
+}
```

Comment:
A lot of really great stuff happening here! Really nice use of method chaining on line 123; great use of a for loop to add to the variable wordScore declared outside of the loop; and really great use of an object's bracket notation to find the key using the incoming string.